libs/experimental/langchain_experimental/data_anonymizer/deanonymizer_matching_strategies.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict
+
+
+def default_matching_strategy(
+    text: str, deanonymizer_mapping: Dict[str, Dict[str, str]]
```

Comment:
maybe we should make some dataclass for `deanonymizer_mapping` as current construction is quite complicated